modules/jwt_auth_refresh/src/Entity/JwtRefreshToken.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace Drupal\jwt_auth_refresh\Entity;
+
+use Drupal\Component\Utility\Random;
+use Drupal\Core\Annotation\Translation;
+use Drupal\Core\Entity\Annotation\ContentEntityType;
+use Drupal\Core\Entity\ContentEntityBase;
+use Drupal\Core\Entity\EntityTypeInterface;
+use Drupal\Core\Field\BaseFieldDefinition;
+use Drupal\jwt_auth_refresh\JwtRefreshTokenInterface;
+use Drupal\user\UserInterface;
+use Firebase\JWT\JWT;
+
+/**
+ * @ContentEntityType(
+ *   id = "jwt_refresh_token",
+ *   label = @Translation("JWT Refresh Token"),
+ *   base_table = "jwt_refresh_token",
+ *   entity_keys = {
+ *     "id" = "id",
```

Comment:
Let's add jti here. That should be unique. Also, do we need to do anything special to get uuid's? Maybe that comes with entity by default?